runners/core-java/pom.xml

```diff
@@ -152,6 +152,13 @@
 
     <!-- test dependencies -->
 
+    <dependency>
+      <groupId>com.esotericsoftware.kryo</groupId>
+      <artifactId>kryo</artifactId>
+      <version>2.21</version>
```

Comment:
There shoudn't be any Kryo dependency in runners core. If at all, then this test should be in the Apex runner.